### PR TITLE
Edit existing card

### PR DESCRIPTION
**Pull request by filwas**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -4,29 +4,28 @@ import logo from "../public/FischLogo.svg";
 
 import styles from "./App.module.css";
 import { Card } from "./components/Card";
-import { useState } from "react";
+import { useRef, useState } from "react";
 
 function App() {
+  const [cardsAmount, setCardsAmount] = useState(1)
   const [cards, setCards] = useState([{}]);
```
Comment (Sumick) [inline src/App.tsx:11]: **nitpick:** Why empty object as initial value? Empty array `[]` would be enough

**Review by Sumick — APPROVED** [review]

File: src/App.tsx (hunk 1 above)
Comment (filwas) [reply src/App.tsx:11]: One of the requirements is to display at least one card on the first boot, this is that card. :)

Comment (Sumick) [reply src/App.tsx:11]: But why it does not have any values?

Comment (filwas) [reply src/App.tsx:11]: What do you mean, no values? The card has values, but it only gets created on lines 18-22, where I create a card component for each "empty object" in the cards array. I'll change its name so it's less confusing.

**Pull request merged**